docs/contracts/v3/reference/Deployments.md

```diff
@@ -45,16 +51,16 @@ getPool("0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48", "0xc02aaa39b223fe8d0a0e5c4
 
 The Uniswap Protocol supports trading of ERC20 tokens. In order to swap a native asset like ETH (or MATIC on Polygon), the Uniswap protocol wraps these assets in an ERC20 wrapped native token contract. The protocol uses the following WETH9 addresses on Ethereum and WMATIC addresses on Polygon.
```

Comment:
Unrelated to the change, but this is super useful info! We should add it to the v3-sdk guides at some point. I wish I knew this from the start 😄